portfolio.html

```diff
@@ -4,8 +4,42 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+    />
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <div class="brand">
+        <h1 class="brand-name">Brand Name</h1>
+      </div>
+      <div class="hamburger-btn">
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
+    </header>
+    <main>
+      <section class="headline">
+        <h1>Hello, I am Yaseen Akbari</h1>
+        <p>
+          I'm a full-stack web developer! I can help you improve your business.
+          have designed and developed websites for both large sized and small
+          sized business owners from domains like eCommerce, NGO, legal,
+          medical, finance, and many more.
+        </p>
+        <p>If you are interested, please feel free to contact me.</p>
+        <h2>LET’S CONNECT</h2>
+        <div class="social-media">
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+          <a href="#"><i class="fa-brands fa-square-facebook"></i></a>
+        </div>
```

Comment:
This content should be wrapped in a `<ul>` tag with each icon wrapped in a`<li>` tag.

<img width="608" alt="Screen Shot 2022-08-18 at 8 44 09" src="https://user-images.githubusercontent.com/82426682/185410188-abf6996f-2c9c-46ad-ae49-ed68d7e3202d.png">